Missionframework/modules/04_build/fnc/fn_build_handleMouse.sqf

```diff
@@ -6,18 +6,18 @@
     File: fn_build_handleMouse.sqf
     Author: KP Liberation Dev Team - https://github.com/KillahPotatoes
     Date: 2018-09-09
-    Last Update: 2018-10-08
+    Last Update: 2018-11-04
     License: GNU General Public License v3.0 - https://www.gnu.org/licenses/gpl-3.0.html
 
     Description:
-    Handle build display mouse movement events
+        Handle build display mouse movement events
 
     Parameter(s):
-        0: STRING - Mouse move event
-        1: ARRAY - Additional event arguments
+        _mode   - Mouse event name              [STRING]
+        _args   - Additonal event parameters    [ANY]
```

Comment:
ARRAY, defaults to nil

_args just accepts an array, so `["myMode", "myArgs"] call KPLIB_fnc_build_handleMouse;` would (should) create an error.